main.c

```diff
@@ -31,7 +31,8 @@
 
 #if SEXP_USE_MAIN_HELP
 void sexp_usage(int err) {
-  printf("usage: chibi-scheme [<options> ...] [<file> <args> ...]\n"
+  FILE *out = err ? stderr : stdout;
```

Comment:
can you just inline this?  chibi is written for very old compilers which don't allow var declarations mid-function.

Reply:
I can, but isn't that declaration at the very beginning of the function?

Reply:
I thought a separate declaration is clearer since there is some `#ifdef` magic in the middle of the `printf()` call, making it quite complex.

Reply:
Oh, sorry, nm.